src/core.js

```diff
@@ -67,8 +68,8 @@ var QUnit,
 	objectValues = function( obj ) {
 		var key, val,
 			vals = QUnit.is( "array", obj ) ? [] : {};
-		for ( key in obj ) {
-			if ( hasOwn.call( obj, key ) ) {
+		for (key in obj) {
+			if (hasOwn.call( obj, key )) {
```

Comment:
something is breaking the code style, and it's becoming hard to check the real diffs here.